blocks/api/raw-handling/shortcode-converter.js

```diff
@@ -25,33 +25,37 @@ export default function( HTML ) {
 			return acc;
 		}
 
+		const transformTags = transform.tag instanceof Array ? transform.tag : [ transform.tag ];
```

Comment:
Lodash's `_.castArray` could be handy here: https://lodash.com/docs/4.17.4#castArray

Reply:
See also: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#instanceof_vs_isArray